src/index.tsx

```diff
@@ -53,7 +53,9 @@ const CSVReader: React.FC<CSVReaderProps> = ({
             encoding: fileEncoding,
           }),
         )
-        onFileLoaded(csvData.data, fileInfo)
+        if(csvData) {
+          onFileLoaded(csvData.data, fileInfo);
+        }
```

Comment:
```suggestion 
          onFileLoaded(csvData?.data ?? [], fileInfo);
```

Reply:
The interface reports that onFileLoaded should return an array, so the fallback should be an array.
Doing nothing could be bad for UX and dev experience.

Errors are threated in the onError method.